example-2/evm/src/contracts/lendingHub/HubGetters.sol

```diff
@@ -78,165 +78,15 @@ contract HubGetters is Context, HubStructs, HubState {
         return _state.totalAssets[assetAddress];
     }
-
-    function normalizeAmount(uint256 denormalizedAmount, uint256 interestAccrualIndex) public view returns (uint256) {
-        return (denormalizedAmount * _state.interestAccrualIndexPrecision) / interestAccrualIndex;
-    }
-
-    function denormalizeAmount(uint256 normalizedAmount, uint256 interestAccrualIndex) public view returns (uint256) {
-        return (normalizedAmount * interestAccrualIndex) / _state.interestAccrualIndexPrecision;
-    }
-
-    function denormalizeVaultAmount(VaultAmount memory va, address assetAddress) internal view returns (VaultAmount memory) {
-        AccrualIndices memory indices = getInterestAccrualIndices(assetAddress);
-        uint256 denormalizedDeposited = denormalizeAmount(va.deposited, indices.deposited);
-        uint256 denormalizedBorrowed = denormalizeAmount(va.borrowed, indices.borrowed);
-        return VaultAmount({
-            deposited: denormalizedDeposited,
-            borrowed: denormalizedBorrowed
-        });
-
-    }
-
-    function getOraclePrices(address assetAddress) internal view returns (uint64) {
-        AssetInfo memory assetInfo = getAssetInfo(assetAddress);
-
-        IMockPyth.PriceFeed memory feed = mockPyth().queryPriceFeed(assetInfo.pythId);
-
-        // sanity check the price feeds
-        require(feed.price.price > 0, "negative prices detected");
-
-        // Users of Pyth prices should read: https://docs.pyth.network/consumers/best-practices
-        // before using the price feed. Blindly using the price alone is not recommended.
-        return uint64(feed.price.price);
-    }
-
-    function getVaultEffectiveNotionals(address vaultOwner) internal view returns (uint256, uint256) {
-
-        uint256 effectiveNotionalDeposited = 0;
-        uint256 effectiveNotionalBorrowed = 0;
-
-        address[] memory allowList = getAllowList();
-
-        for(uint i=0; i<allowList.length; i++) {
-            address asset = allowList[i];
-
-            VaultAmount memory normalizedAmounts = getVaultAmounts(vaultOwner, asset);
-
-            uint64 price = getOraclePrices(asset);
-            
-            AssetInfo memory assetInfo = getAssetInfo(asset);
-
-            AccrualIndices memory indices = getInterestAccrualIndices(asset);
-            
-            uint256 denormalizedDeposited = denormalizeAmount(normalizedAmounts.deposited, indices.deposited);
-            uint256 denormalizedBorrowed = denormalizeAmount(normalizedAmounts.borrowed, indices.borrowed);
-
-            effectiveNotionalDeposited += denormalizedDeposited * price / (10**assetInfo.decimals);
-
-            effectiveNotionalBorrowed += denormalizedBorrowed * assetInfo.collateralizationRatio * price / (10**assetInfo.decimals * getCollateralizationRatioPrecision());
-      
-        }    
-
-        return (effectiveNotionalDeposited, effectiveNotionalBorrowed);
-    }
-
-
-    // TODO: cycle through all assets in the vault
```

Comment:
do you mind just adding back all the TODOs? or at least consolidating them somewhere? helpful reminders for me